apps/prairielearn/src/pages/studentAssessmentInstance/studentAssessmentInstance.sql

```diff
@@ -1,3 +1,11 @@
+-- BLOCK select_assessment_instance
+SELECT
+  ai.*
+FROM
+  assessment_instances AS ai
+WHERE
+  ai.id = $assessment_instance_id;
+
 -- BLOCK select_instance_questions
```

Comment:
Nit: there's a change in how `prev_title` is computed. Before, it would match what's displayed in the question list (e.g. "Question 2" for exams, "WS2.1" for homeworks). Now, it would show "Question 2" and "Question WS2.1". This is very much a nit, but capital-Q "Question WS2.1" feels off to me. I'm not yet requesting changes, just raising this for discussion.

Reply:
Good catch. I'll add it to the batch. From what I could get, this is for forced order help text.

Reply:
Changed in 34cb0f1f1. There was no reason for "Question" to be part of the query, I moved it to the partial.

Reply:
Note that "Question ###" (with a capital letter) is how exam questions are actually identified:

<img width="439" alt="Screenshot 2024-01-03 at 15 07 20" src="https://github.com/PrairieLearn/PrairieLearn/assets/1476544/54da0664-82d9-43d2-b88a-702ae23fdcaa">

I'm sure students will be able to tell that "question 1" and "Question 1" are the same thing 🙃 but the two ways that `prev_title` was previously being computed were very intentional.

Reply:
I have changed it to have the same behaviour as before this PR. Changes can be discussed in the future.